src/tests/test_claim_detector.py

```
@@ -0,0 +1,254 @@
+"""Unit tests for the AI-driven claim detection system."""
+
+import pytest
+from unittest.mock import AsyncMock, patch
```

Ruff (0.11.9)

`unittest.mock.AsyncMock` imported but unused

Remove unused import

(F401)